packages/manager/src/queries/preferences.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  getUserPreferences,
+  updateUserPreferences,
+  UserPreferences,
+} from '@linode/api-v4/lib/profile';
+import { APIError } from '@linode/api-v4/lib/types';
+import { useMutation, useQuery } from 'react-query';
+import { ManagerPreferences } from 'src/types/ManagerPreferences';
+import { queryClient, queryPresets } from './base';
+
+export const queryKey = 'preferences';
+
+export const usePreferences = (enabled = true) =>
+  useQuery<UserPreferences, APIError[]>(queryKey, getUserPreferences, {
+    ...queryPresets.oneTimeFetch,
+    enabled,
+  });
+
+export const useMutatePreferences = () => {
+  const { data: preferences } = usePreferences();
+
+  return useMutation<
+    ManagerPreferences,
+    APIError[],
+    Partial<ManagerPreferences>
+  >((data) => updateUserPreferences({ ...(preferences ?? {}), data }), {
```

Comment:
Oops, I think I messed you up here. The data value should be spread.

```suggestion
  >((data) => updateUserPreferences({ ...(preferences ?? {}), ...data }), {
```